liferay-workspace/tests/src/test/groovy/com/liferay/blade/tests/BladeTests.groovy

```diff
@@ -0,0 +1,132 @@
+package com.liferay.blade.tests
+
+import org.gradle.testkit.runner.GradleRunner
+import static org.gradle.testkit.runner.TaskOutcome.*
+import spock.lang.Specification
+import de.undercouch.gradle.tasks.download.Download;
+import java.util.jar.JarInputStream;
+import java.io.FileNotFoundException;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Request.Builder;
+
+class BladeTests extends Specification {
+	def setup () {
+		def initBundle = GradleRunner.create()
+			.withProjectDir(new File(projectDir))
+			.withArguments('initBundle')
+			.build()
+
+		def bladeclijar = 'com.liferay.blade.cli_1.0.1.201606142156.jar'
+		def url = 'http://releases.liferay.com/tools/blade-cli/1.0.1.201606142156/plugins/${bladeclijar}'
+		def file = new File("${bladeclijar}")
+
+		if (!file.isFile()) {
+			file.newOutputStream()
+			file << URL(url).openStream()
+			file.close()
+		}
+
+		if (!file.isFile()) {
+			throw new GradleException(new FileNotFoundException());
+		}
+
+		public void executeBlade(arg1='', arg2='', arg3='', arg4='') {
```

Comment:
lets define this outside of this test method.  put it in the body of the BladeTests outside of this method.